opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceHeader.jsx

```diff
@@ -221,81 +222,10 @@ const WorkspaceHeader = ({
             <WorkspacePopover workspace={workspace} />
           </div>
         </Security>
-        {variant === 'dashboard' && (
+        {variant === 'dashboard' && !isFABReplaced && (
           <Security
             needs={[EXPLORE_EXUPDATE, INVESTIGATION_INUPDATE]}
             hasAccess={canEdit}
-            placeholder={
```

Comment:
Security placeholder was the same as the child component, so either the Security is irrelevant or we need to remove the placeholder.